db/migrations/006_add_column_name_to_attributes.rb

```diff
@@ -0,0 +1,23 @@
+Sequel.migration do
+  up do
+    alter_table(:attributes) do
+      add_column :column_name, String
+    end
+
+    Magma.instance.db.execute("UPDATE attributes SET column_name=attribute_name")
+
+    alter_table(:attributes) do
+      set_column_not_null :column_name
```

Comment:
We'll add the `column_name` column and backfill existing rows before making the column not nullable.